std_msgs/Header header
geometry_msgs/PoseArray pose_array
float32[] angles
int32 trigger_flag  